### PR TITLE
Addresses issue #1 and #2 issues 

### LINKED ISSUE #1: We should raise an error if multiple faces are available.
https://github.com/openabis/openabis-ageitgey-face-recognition/blob/f93731359f3930f0f4aaba02f3df20818ab99101/openabis_ageitgey_face_recognition/__init__.py#L22

### LINKED ISSUE #2: MATCHING_TOLERANCE and DEFAULT_ENCODING values should be coming from the config
https://github.com/openabis/openabis-ageitgey-face-recognition/blob/f93731359f3930f0f4aaba02f3df20818ab99101/openabis_ageitgey_face_recognition/__init__.py#L9

### PR DESCRIPTION
**Pull request by renceInbox**
Fix #2 
- Remove matching tolerance variable but retained the encoding name and rename it to IMAGE_ENCODING. This is not dependent on the config.

Fix #1 
- Added exceptions if multiple faces found in the image.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,4 +12,7 @@
     install_requires=[
         'face-recognition',
     ],
+    package_data={
```
Comment (jeremi) [inline setup.py:15]: What is the effect of this? Why is this necessary to fix those issues?

**Review by jeremi — CHANGES_REQUESTED** [review]
the PYC files are not necessary I think.

**Comment by renceInbox** [comment]
Refer to #4 and #5
